app/assets/javascripts/discourse/app/controllers/edit-slow-mode.js

```diff
@@ -67,8 +67,13 @@ export default Controller.extend(ModalFunctionality, {
   },
 
   @discourseComputed("hours", "minutes", "seconds")
-  submitDisabled(hours, minutes, seconds) {
-    return this.saveDisabled || !(hours || minutes || seconds);
+  durationIsSet(hours, minutes, seconds) {
+    return hours || minutes || seconds;
+  },
```

Comment:
great, now I think you can use `import { or } from "@ember/object/computed";``

```
durationIsSet: or("hours", "minutes", "seconds"),
```

I think it should work